library/eclipse-mosquitto

```diff
@@ -1,7 +1,11 @@
 Maintainers: David Audet <david.audet@ca.com> (@daudetCA)
 GitRepo: https://github.com/eclipse/mosquitto.git
 
-Tags: 1.4.10, latest
+Tags: 1.4.12, latest
+GitCommit: 0bb602ed7a89da80d25e6b959a130fdcf0556be5
+Directory: docker/1.4.12
+
+Tags: 1.4.10
```

Comment:
So the tag will not be deleted, but users won't know that 1.4.10 is still available? I guess that's ok as long as people using 1.4.10 can continue to do so...